google/cloud/compute/v1/compute.proto

```diff
@@ -559,11 +558,6 @@ message AllocationSpecificSKUAllocationReservedInstanceProperties {
   // Specifies type of machine (name only) which has fixed number of vCPUs and fixed amount of memory. This also includes specifying custom machine type following custom-NUMBER_OF_CPUS-AMOUNT_OF_MEMORY pattern.
   string machine_type = 227711026;
 
-  // Specifies whether this VM may be a stable fleet VM. Setting this to "Periodic" designates this VM as a Stable Fleet VM.
-  //
-  // See go/stable-fleet-ug for more details.
-  MaintenanceInterval maintenance_interval = 134932593;
```

Comment:
They really did get rid of this? OK if so, just checking it's not something we omitted by mistake.

Reply:
Looks like they did. There were not any changes in the converter which might have caused something like sporadic removal of fields.